src/Widgets/IconWidget/IconWidgetComponent.js

```diff
@@ -7,9 +7,11 @@ function IconWidgetComponent({ widget }) {
   const link = widget.get("link");
   const size = widget.get("size");
 
-  if (["center", "right"].includes(widget.get("alignment"))) {
+  const outerClassName = alignmentClassName(widget.get("alignment"));
```

Comment:
* [ ] FYI Since there is no `innerClassName`, how about `wrapperClassName`? Alternatively, `className` may be good enough, as it's obvious what it's used for.